sdk/tables/azure-data-tables/azure/data/tables/_generated/_azure_table.py

```diff
@@ -28,7 +28,7 @@ class AzureTable(object):
     :vartype table: azure.data.tables.operations.TableOperations
     :ivar service: ServiceOperations operations
     :vartype service: azure.data.tables.operations.ServiceOperations
-    :param url: The URL of the service account or table that is the target of the desired operation.
+    :param url: The URL of the service account or table that is the targe of the desired operation.
```

Comment:
lmao, not super high priority but can you pinig the service team about the swagger / add a fix for this

Reply:
Thank you for finding this, making a PR now.